Files/Views/LayoutModes/GridViewBrowser.xaml

```diff
@@ -172,7 +174,7 @@
                     x:Load="False">
                     <TextBox
                         x:Name="GridViewTextBoxItemName"
-                        Width="{Binding FolderSettings.GridViewSize, ElementName=PageRoot, Mode=OneWay}"
+                        Width="{Binding ElementName=GridViewBrowserListedItem, Path=(tui:FrameworkElementExtensions.ActualWidth)}"
                         Margin="0"
```

Comment:
```suggestion
                        Background="{ThemeResource SolidBackgroundFillColorSecondary}"
                        Margin="0"
```
the selection indicator and other items under the text box shouldn't be visible, would you mind making this change to fix that?
![image](https://user-images.githubusercontent.com/59544401/128553091-7a23c07c-0407-46b2-bc87-20081e131683.png)

Reply:
Looks good 👍